Packages/DesignSystem/Sources/DesignSystem/SFSymbols.swift

```diff
@@ -0,0 +1,34 @@
+
+
+import Foundation
+import SwiftUI
+
+// Functions to cope with extending SF symbols
+// images named in lower case are Apple's symbols
+// images inamed in CamelCase are custom
+
+extension Image {
+  
+  public init(imageNamed: String) {
+    if imageNamed.lowercased() == imageNamed {
```

Comment:
I wish we could avoid that or do it differently. This could be a bottle neck to to this str compare for each icons of the status row actions bar. It might seems nothing but when it's supposed to scroll at 120 FPS this could have an impact :/

Reply:
All good now